src/TodoList.js

```diff
@@ -1,35 +1,20 @@
 import React from 'react';
+import TodoListItem from './TodoListItem';
 
-const todoList = [
-    {
-      id: 1,
-      title: "Complete React assignment",
-    },
-    { id: 2,
-      title: "Do laundry",
-    },
-    {
-      id: 3,
-      title: "Walk the dog",
-    }
-  ]
 
-function TodoList () {
-    return(
-        <ul>      
-        {todoList.map(function(item){
-          return (
-            <li key={item.id}>
-              <span>
-                {item.title}
-              </span>
-            </li>
-          )
-        })}
-      </ul>
-    )
 
-}
+const TodoList = props => {
+  const {todoList} = props;
+	return (
+		<div>
+        <ul >
+				{todoList.map((todo) => {
+          return <TodoListItem todo={todo} />
```

Comment:
This isn't necessary for functionality, but adding a key attribute on any mapped element is a good practice for performance! If you run the app in a browser and open the console, you'll see a warning message about this. It'd be awesome if you'd update the code so the warning message no longer appears!